deploy/ec2/tool_jet_ubuntu_bionic.pkr.hcl

```diff
@@ -8,7 +8,7 @@ packer {
 }
 
 source "amazon-ebs" "ubuntu" {
-  ami_name      = "tooljet_latest_ubuntu_bionic"
+  ami_name      = "tooljet_v0.5.11.ubuntu_bionic"
```

Comment:
Can we generate `ami_name` using the `TOOLJET_VERSION` set in `config/application.rb:22` ?

Reply:
Nope, we can pass it as an argument to `packer build`. That makes more sense.